src/renderer/components/dock/edit-resource/view.tsx

```diff
@@ -123,8 +123,12 @@ class NonInjectedEditResource extends React.Component<EditResourceProps & Depend
     );
   }
 
+  @computed get configurationInView() {
+    return this.props.editResourceStore.getData(this.tabId)?.draft;
+  }
+
```

Comment:
This computed is crucial. `this.tabData` or `tabData` from `save()` are not correct. Again, this is kludge because that's simply not OK..

Reply:
With testing (and reading code) your claim about `computed` is incorrect. Since this value is only read in one place (essentially an action) it does not need to be computed.

Reply:
`@computed` maybe, since `getData` already reads from reactive sure. So maybe I should have said "This getter is crucial", since `get tabData()` cannot be used.